make-it/EmMakefile

```diff
@@ -72,7 +72,7 @@ EMCC= emcc
 EM_OPTFLAG = -O3
 
 ifeq ($(BUILD), debug)
-   EM_OPTFLAG = -g4 -s ASSERTIONS=2 -s SAFE_HEAP=1
+   EM_OPTFLAG = -g -s ASSERTIONS=2 -s SAFE_HEAP=1
```

Comment:
Looks like [-g4 was removed](https://github.com/emscripten-core/emscripten/blame/a8a0d8e79ed5d4aa640e18ae7497ef4c48e5beaa/ChangeLog.md#L149) as it is the same as -g3 (and we don't need the source maps `-gsource-map` feature if we have the dwarf symbols). In addition [-g is the same as -g3](https://emscripten.org/docs/tools_reference/emcc.html#emcc-g).